modules/client-configuration/pages/autoinst-distributions.adoc

```diff
@@ -0,0 +1,100 @@
+For information, see xref:client-configuration:autoinst-distributions.adoc#based-on-iso-image[Distribution Based on an ISO Image].
+
+Alternatively, you can install [package]``tftpboot-installation`` package.
+For information, see xref:client-configuration:autoinst-distributions.adoc#based-on-rpm-package[Distribution Based on a RPM Package].
+
+You must also have a base channel synchronized on your {productname} Server for the same operating system version as those files.
+
+When you have the files ready, and the base channel synchronized, you need to declare the distribution.
+This operation associates the installation files to the base channel.
+The distribution can be referred to by one or more installation profiles.
+For information, see xref:client-configuration:autoinst-distributions.adoc#declare-distribution[Declare an Autoinstallable Distribution].
+
+
+[[based-on-iso-image]]
+== Distribution Based on an ISO Image
+
+This method assumes you have installation media for the operating system you want to install on the clients. 
+This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [path]``initrd`` file, and other files required to boot the operating system in installation mode.
+
+
+
+.Procedure: Importing Files from Installation Media
+. Copy the installation media to your {productname} Server.
+  For {suse} operating systems, you can download installation media from https://www.suse.com/download/.
+. Loop-mount the ISO image, and copy its contents somewhere:
+----
+# mount -o loop,ro <image_name>.iso /mnt
+# mkdir -p /srv/www/distributions
+# cp -a /mnt /srv/www/distributions/<image_name>
+# umount /mnt
+----
++
+  Take a note of the file path.
+  You will need it when you declare the distribution to {productname}.
+
+
+[[based-on-rpm-package]]
+== Distribution Based on a RPM Package
+
+This method works on {suse} systems.
+It is simpler than importing contents from an installation media, because it uses prepackaged files for your installation system.
+
+
+
+.Procedure: Extracting Files from an Installation Package
+. On the {productname} Server, install the package whose name starts with [package]``tftpboot-installation``.
+  You can determine its exact name with the command [command]``zypper se tftpboot-installation``
+. Determine where the installation files went with the command [command]``ls -d /usr/share/tftpboot-installation/*``.
+  Take a note of the file path.
+  You will need it when you declare the distribution to {productname}.
+
+This procedure prepares the installation of the same operating system version as the one that powers your {productname} Server.
+If you want to install a different operating system or version on the client, you need to manually get the package [package]``tftpboot-installation`` from the distribution it belongs to.
+You will find it on the {productname} Server in the products you mirrored with the command [command]``find /var/spacewalk -name "tftpboot-installation-*"``.
```

Comment:
The "find" command might be a bad idea. It can take endless when a lot of products were mirrored.

Better use the WebUI and check the main channels (SLES-Pool/Update for SLE12 or SLE-Module-Basesystem for SLE15). The Package Search functionallity can also help.
When the package is found, check the details. It show also the local path below /var/spacewalk/

Reply:
Also add a note which tell the customer that they need to take care for updating these packages.

Reply:
The "find" command can be pretty fast, even with many files. It's not a "grep".

But ok, let's indicate the SUSE Manager way. Here is the new version:
```
In the [menu]``Package Search`` input box {productname}, search the packages whose names start with [package]``tftpboot-installation``, then look at the package's details. They show the local path below [path]``/var/spacewalk/``.
```